src/index.html

```diff
@@ -5,9 +5,72 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card body__card" data-qa="card">
+      <div class="card__picture">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+      </div>
+
+      <div class="card__desription">
+        <h1 class="card__mark">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h1>
+      </div>
```

Comment:
1. No need in this wrapper
2. Also, you can have only 1 `<h1>` element per page, but you are going to reuse `.card` block. Think of using `<h3>`